files/__main__.py

```diff
@@ -170,6 +170,7 @@ def before_request():
 		return 'Please use a "User-Agent" header!', 403
 
 	ua = g.agent.lower()
+	g.debug = bool_from_string(app.config["DEBUG"])
```

Comment:
We use `app.debug`, `app.config["DEBUG"]`, and `g.debug` all essentially interchangeably. Perhaps use `app.debug` here to be consistent with L97 and standardize around the `{app,g}.debug` variants.